templates/home.html

```diff
@@ -1,17 +1,13 @@
 {{ define "content" }}
+  {{ include "partials/flashes" }}
+  
+  <h1 class="sm:text-5xl md:text-6xl">
+    <span class="text-indigo-600">Go</span>ogle Scraper
+  </h1>
 
-<h1 class="text-primary">HELLO GOLANG</h1>
-{{ include "partials/flashes" }}
-
-{{ if .data.authenticatedUser }}
-  <h2 class="greeting">Welcome {{ .data.email }}</h2>
-  <h2><a href="/keyword">Keyword</a></h2>
-  <form class="mt-8 space-y-6" action="/logout" method="POST">
-    <button type="submit">Logout</button>
-  </form>
-{{ else }}
-  <h2><a href="/login">Login</a></h2>
-  <h2><a href="/register">Register</a></h2>
-{{ end }}
+  <h4 class="display-subtitle sm:mt-5 md:text-xl">
+    Gathering Google search information with your own keywords
+  </h4>
 
+  <a href="/keyword"><button class="button-primary sm:mt-8 sm:text-lg sm:px-8 sm:py-4">Get Started</button></a>
```

Comment:
Same as above to move the styles to the screen sctylesheet

Reply:
Fixed in a6e038f935ccaff5800ed8f2e6ddcfd998182677